apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QIntegrationTestBase.java

```diff
@@ -490,9 +500,25 @@ protected List<Map<String, String>> insertRows(
 
   protected List<Map<String, Object>> insertTypedRows(
       String keyspaceName, String tableName, List<Map<String, Object>> rows) {
+    return insertTypedRowsInternal(keyspaceName, tableName, rows, null);
+  }
+
+  protected List<Map<String, Object>> insertTypedRows(
+      String keyspaceName,
+      String tableName,
+      List<Map<String, Object>> rows,
+      Boolean compactMapData) {
```

Comment:
Should probably pass primitive `boolean`, no need to pass `null`?

Reply:
There is a null check here in `getInsertRowRequest` method, where the request query param is either set or not set based on this Boolean wrapper object.